frontend/src/metabase/meta/types/Parameter.js

```diff
@@ -10,6 +10,7 @@ export type ParameterType = string;
 
 // a URL-safe encoding of a parameter value
 export type ParameterValue = string;
+export type ParameterValueOrArray = string | Array<string>;
```

Comment:
I had to add this extraneous type over just adding the Array type to `ParameterValue` due to flow blowing up in other parts of `frontend/src/metabase/meta/Parameter.js`. I kind of hate this so I'm very much open to suggestions.